patches/coreclr/0001-wip.patch

```diff
@@ -0,0 +1,28 @@
+From 399521c59b59be0807137d3c1763ad71c5677b9b Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Mon, 19 Nov 2018 11:18:43 -0600
+Subject: [PATCH] wip
+
+---
+ global.json | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/global.json b/global.json
+index b69f6376c6..abc013309f 100644
+--- a/global.json
++++ b/global.json
+@@ -7,7 +7,7 @@
+     "python": "2.7.15"
+   },
+   "msbuild-sdks": {
+-    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.18552.21",
+-    "Microsoft.DotNet.Helix.Sdk": "1.0.0-beta.18552.21"
++    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.18556.6",
++    "Microsoft.DotNet.Helix.Sdk": "1.0.0-beta.18556.6"
```

Comment:
Would it be reasonable to use global.json updates for this rather than patches?

Reply:
Changing these doesn't seem necessary. Arcade SDK overrides no matter what version, and the Helix package doesn't even seem to be used as it's not in the prebuilt report.